persistence-modules/spring-data-jpa/pom.xml

```diff
@@ -55,14 +64,14 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
-        <!-- JUnit platform launcher -->
-        <!-- To be able to run tests from IDE directly -->
+        JUnit platform launcher
```

Comment:
Let's not have stray words in the pom here. Once they uncomment, then this file will no longer parse.